maintainers/scripts/update-ruby-packages

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p bundler bundix
+
+set -euf -o pipefail
+
+(
+  cd pkgs/development/ruby-modules/with-packages
+  rm -f gemset.nix Gemfile.lock
+  bundle lock
+  bundix
```

Comment:
`bundix -l` ;)

Reply:
`bundix -l` is not as reliable as I want it to be, and doesn't add that much of a speedup given that all gems are cached on disk anyway.